packages/contracts/contracts/default-apps/ETHBalanceRefundApp.sol

```diff
@@ -13,27 +12,28 @@ contract ETHBalanceRefundApp {
     uint256 threshold;
   }
 
-  function resolve(bytes memory encodedState, Transfer.Terms memory terms)
-    public
+  function resolve(bytes calldata encodedState)
+    external
```

Comment:
Again with `external` and `calldata`. Can you make a definitive call on this and add it to the specs, please?